### ISSUE TITLE
Assembly version in published 2.14.1 package is wrong

### ISSUE THREAD

**Issue by enovales** [fscheck/FsCheck#513]
All of the published assemblies in the 2.14.1 nuget package seem to have the file version set to "1.0.0.0", which is causing errors in the assembly loader when trying to use it.

I have a reproducible test case here: https://github.com/enovales/FsCheckAssemblyLoadFailure

This project was created with the [MiniScaffold](https://github.com/TheAngryByrd/MiniScaffold). The only modifications I made were to add the FsCheck and Expecto.FsCheck references, remove the `net461` configuration in the solution, and to add some code in the test project to use FsCheck. The tests run fine if you use the FsCheck 2.14.0 package, but fail to load the assembly if you use FsCheck 2.14.1.

**Comment by kurtschelfthout** [comment]
Ok, thanks for the repro. Let me take a look. A priori that is surprising on two counts: the AssemblyInfo files have version 2.14.1.0 and FileVersion has been changing every release since time started; and secondly the loader has not taken the FileVersion into account. only the AssemblyVersion and even then only when the assembly is signed (which FsCHeck is not). I'll have to do some digging.

**Comment by kurtschelfthout** [comment]
So you are right, it indeed does not work and the version is indeed 1.0.0.0. This is not good for my ego, haha.

I pushed https://github.com/fscheck/FsCheck/commit/5c0a7e8b00623e7523ef4807230270850dc26493 which at least fixes the wrong assembly version. It appears that in dotnet build there is a default AssemblyVersionAttribute generation step which overrides the attributes set in the AssemblyInfo.fs files. 

I have also unlisted version 2.14.1 on NuGet so people should no longer see the bad package.

I have to go now so I can't test if this actually resolves the issue, but I'll do that tomorrow. If it does, then my theory will be that the duplicate AssemblyVersionAttribute in the assembly somehow confuses the loader.

I find it somewhat baffling that the build generated assemblies that have two AssemblyVersionAttributes in them, without any warning or error!
